meta0v2/meta0_client.c

```diff
@@ -28,7 +28,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "meta0_utils.h"
 
 static addr_info_t addr = {{0}};
-static gchar *namespace = NULL;
+static gchar namespace[LIMIT_LENGTH_NSNAME] = {0};
```

Comment:
The usage of the `LIMIT_LENGTH_`  macros is to store the string with its trailing NUL character.
So that the real limit on the length of the information is the macro minus 1.

What do you suggest? A global macro renaming? A fix of the documentation about this?